resources/sts/4.10/openshift_aws_vpce_operator_avo_aws_creds_policy.json

```diff
@@ -14,13 +14,17 @@
         "ec2:AuthorizeSecurityGroupIngress",
         "ec2:AuthorizeSecurityGroupEgress",
         "ec2:DescribeSecurityGroupRules",
+        "ec2:DescribeVpcs",
         "ec2:CreateVpcEndpoint",
         "ec2:DeleteVpcEndpoints",
         "ec2:DescribeVpcEndpoints",
         "ec2:ModifyVpcEndpoint",
         "route53:ChangeResourceRecordSets",
         "route53:ListHostedZonesByName",
-        "route53:ListResourceRecordSets"
+        "route53:ListResourceRecordSets",
+        "route53:ListTagsForResource",
+        "route53:CreateHostedZone",
```

Comment:
@tonytheleg do we need `route53:CreateHostedZone` ? It looks like we [create](https://github.com/openshift/aws-vpce-operator/blob/b1a65a2c26d34e77ae3f30279aa8bad7db4170c4/controllers/vpcendpoint/validation.go#L279) a hosted zone if its not found is that required? What is AddtlHostedZoneName?

cc/ @dustman9000

Reply:
The `AddtlHostedZoneName` is new feature add to address [OSD-13509](https://issues.redhat.com/browse/OSD-13509). Details are in that card but essentially we need to be able to create HostedZones for domains for endpoints that wont exist in a customer account in order to properly route traffic to endpoints like OCM in FedRAMP as none of the endpoints are publically available. [OSD-13510](https://issues.redhat.com/browse/OSD-13510) follows up on it to be able to create CNAMES in the additional hosted zone to resolve addresses in the added zone to the correct VPC endpoints,